apps/admin_buildpack_lifecycle.go

```diff
@@ -271,7 +271,7 @@ exit 1
 
 		Expect(push).To(Exit(0))
 		appOutput := cf.Cf("app", appName).Wait()
-		Expect(appOutput).To(Say("buildpacks?:.*\\n.+\\n.*Simple"))
+		Expect(appOutput).To(Say("buildpacks?:.*\\n.+\\n.+\\n.*Simple"))
```

Comment:
We might as well just match any number of new lines?
```suggestion
		Expect(appOutput).To(Say("(?:s)buildpacks?:.*Simple"))
```